pkg/cmd/prebuild/update.go

```diff
@@ -19,10 +18,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	branchFlag         string
+	retentionFlag      int
+	commitIntervalFlag int
+	triggerFilesFlag   []string
+	projectConfigFlag  string
```

Comment:
I believe this variable should be removed in regards to https://github.com/daytonaio/daytona/pull/1217#discussion_r1793032622

Reply:
as we are using the variable for the updating of the projectConfig from the arguments, more like getting the arguments and saving in this variable so we can use throughout the code.

Reply:
ok will be updated.